components/game_table/dota2/game_table_character_custom.lua

```diff
@@ -0,0 +1,56 @@
+---
+-- @Liquipedia
+-- wiki=dota2
+-- page=Module:GameTable/Character/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Array = require('Module:Array')
+local CharacterIcon = require('Module:CharacterIcon')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local GameTableCharacter = Lua.import('Module:GameTable/Character')
+
+local NONBREAKING_SPACE = '&nbsp;'
+
+local CustomGameTableCharacter = Class.new(GameTableCharacter)
+
+---@return integer
+function CustomGameTableCharacter:getNumberOfBans()
+	return 7
+end
+
+---@param opponentIndex number
+---@param playerIndex number
+---@return string
+function CustomGameTableCharacter:getCharacterKey(opponentIndex, playerIndex)
+	return 'team' .. opponentIndex .. 'hero' .. playerIndex
+end
+
+---@param game CharacterGameTableGame
+---@param opponentIndex number
+---@param key string
+---@return Html
+function CustomGameTableCharacter:displayCharacters(game, opponentIndex, key)
+	local makeIcon = function(character)
+		return CharacterIcon.Icon{character = character, size = self.iconSize, date = game.date}
+	end
+
+	local icons = Array.map(game[key][opponentIndex] or {}, makeIcon)
+
+	return mw.html.create('td')
+		:node(#icons > 0 and table.concat(icons, NONBREAKING_SPACE) or nil)
```

Comment:
Is the only diff the NBSP vs nospace?

Reply:
Yes

Reply:
I think we can just standardize the display to match other game tables

Reply:
Also the side color is not used here, forgot to say..

Reply:
Hmm, maybe we can make options like showSideClass and iconSeparator.